test/FIFOMarket.test.ts

```diff
@@ -637,7 +888,7 @@ describe('FIFOMarket', () => {
           .toString()
       );
     });
-    it('should correctly pay suppliers when multiple different suppliers removals are used to fulfill an order', async () => {
+    it('correctly pay suppliers when multiple different suppliers removals are used to fulfill an order', async () => {
```

Comment:
```suggestion
    it('should correctly pay suppliers when multiple different suppliers removals are used to fulfill an order', async () => {
```

I think the changed string would better be used as a describe loop, but it/test declarations are better read when they indicate that something is the expected but not guaranteed behavior